### PR TITLE
Radarr Unmonitored/Cinema Release Toggles

### LINKED ISSUE #1480: Movies from Radarr v3 don't show up in the calendar on the homepage
###### Operating System: Docker image (official)
<hr>

##### Problem Description:

I've added the radarr integration for the homepage, and enabled it. When testing the connection, it works. But when checking the calendar on the homepage, movies from the radarr calendar aren't put in the calendar on the homescreen.

When searching media, the movies do show up.

<hr>

##### Reproduction Steps:

Very specific:
* Add radarr to a docker-compose file, like in htpcBeginner/traefik
* Add organizr
* Point organizr to the radarr docker image with internal hostname, and enable the connection
* Check the calendar on the homepage

<hr>

#### Errors on screen?  If so paste here:

```

```

### PR DESCRIPTION
**Pull request by HalianElf**
Should close #1480 and my FR from 2 years ago.

### CODE REVIEW

File: api/homepage/radarr.php [hunk 1]
```diff
@@ -349,17 +361,21 @@ public function formatRadarrCalendar($array, $number, $url)
 		$gotCalendar = array();
 		$i = 0;
 		foreach ($array as $child) {
-			if (isset($child['physicalRelease'])) {
+			if (isset($child['physicalRelease']) || $this->config['radarrCinemaRelease']) {
```
Comment (causefx) [inline api/homepage/radarr.php]: I would change this to:
```
if (isset($child['physicalRelease']) || (isset($child['inCinemas']) && $this->config['radarrCinemaRelease'])) {
```
Edited to leave just the one change....

Comment (HalianElf) [reply api/homepage/radarr.php]: I can do that. I also want to check against those movies in that issue.

**Comment by causefx** [comment]
for sure :)

**Comment by HalianElf** [comment]
The Digital Releases will require the changes for the v3 endpoint so that toggle won't do anything until that. I hard coded the v3 endpoint in mine to test it.

**Pull request merged**